compilation failure with nbee

I'm running Ubuntu 12.04 amd64. I cloned the repo at 31704b16 and ran `./boot.sh&& ./configure --prefix=$PWD/install && make install`, and I see this error:

```
gcc -Wstrict-prototypes  -g -O2 -Wall -Wno-sign-compare -Wpointer-arith -Wdeclaration-after-statement -Wformat-security -Wswitch-enum -Wunused-parameter -Wstrict-aliasing -Wbad-function-cast -Wcast-align -Wstrict-prototypes -Wold-style-definition -Wmissing-prototypes -Wmissing-field-initializers -Wno-override-init -export-dynamic  -o udatapath/ofdatapath udatapath/udatapath_ofdatapath-action_set.o udatapath/udatapath_ofdatapath-crc32.o udatapath/udatapath_ofdatapath-datapath.o udatapath/udatapath_ofdatapath-dp_actions.o udatapath/udatapath_ofdatapath-dp_buffers.o udatapath/udatapath_ofdatapath-dp_control.o udatapath/udatapath_ofdatapath-dp_exp.o udatapath/udatapath_ofdatapath-dp_ports.o udatapath/udatapath_ofdatapath-flow_table.o udatapath/udatapath_ofdatapath-flow_entry.o udatapath/udatapath_ofdatapath-group_table.o udatapath/udatapath_ofdatapath-group_entry.o udatapath/udatapath_ofdatapath-match_std.o udatapath/udatapath_ofdatapath-meter_entry.o udatapath/udatapath_ofdatapath-meter_table.o udatapath/udatapath_ofdatapath-packet.o udatapath/udatapath_ofdatapath-packet_handle_std.o udatapath/udatapath_ofdatapath-pipeline.o udatapath/udatapath_ofdatapath-udatapath.o lib/libopenflow.a oflib/liboflib.a oflib-exp/liboflib_exp.a nbee_link/libnbee_link.a  -ldl -lnbee 
nbee_link/nbee_link.cpp:70: error: undefined reference to 'operator new(unsigned long)'
nbee_link/nbee_link.cpp:355: error: undefined reference to 'std::basic_string<char, std::char_traits<char>, std::allocator<char> >::basic_string(char const*, std::allocator<char> const&)'
nbee_link/nbee_link.cpp:356: error: undefined reference to 'std::basic_string<char, std::char_traits<char>, std::allocator<char> >::basic_string(char const*, std::allocator<char> const&)'
nbee_link/nbee_link.cpp:358: error: undefined reference to 'std::basic_string<char, std::char_traits<char>, std::allocator<char> >::compare(char const*) const'
nbee_link/nbee_link.cpp:369: error: undefined reference to 'std::basic_string<char, std::char_traits<char>, std::allocator<char> >::compare(char const*) const'
nbee_link/nbee_link.cpp:380: error: undefined reference to 'std::basic_string<char, std::char_traits<char>, std::allocator<char> >::compare(char const*) const'
nbee_link/nbee_link.cpp:390: error: undefined reference to 'std::basic_string<char, std::char_traits<char>, std::allocator<char> >::compare(char const*) const'
/usr/include/c++/4.6/bits/basic_string.h:235: error: undefined reference to 'std::basic_string<char, std::char_traits<char>, std::allocator<char> >::_Rep::_S_empty_rep_storage'
/usr/include/c++/4.6/bits/basic_string.h:235: error: undefined reference to 'std::basic_string<char, std::char_traits<char>, std::allocator<char> >::_Rep::_S_empty_rep_storage'
/usr/include/c++/4.6/bits/basic_string.h:244: error: undefined reference to 'std::basic_string<char, std::char_traits<char>, std::allocator<char> >::_Rep::_M_destroy(std::allocator<char> const&)'
/usr/include/c++/4.6/bits/basic_string.h:244: error: undefined reference to 'std::basic_string<char, std::char_traits<char>, std::allocator<char> >::_Rep::_M_destroy(std::allocator<char> const&)'
nbee_link/nbee_link.cpp:542: error: undefined reference to 'std::basic_string<char, std::char_traits<char>, std::allocator<char> >::~basic_string()'
nbee_link/nbee_link.cpp:542: error: undefined reference to 'std::basic_string<char, std::char_traits<char>, std::allocator<char> >::~basic_string()'
/usr/include/c++/4.6/ext/new_allocator.h:98: error: undefined reference to 'operator delete(void*)'
/usr/include/c++/4.6/ext/new_allocator.h:92: error: undefined reference to 'operator new(unsigned long)'
/usr/include/c++/4.6/bits/stl_tree.h:973: error: undefined reference to 'std::_Rb_tree_insert_and_rebalance(bool, std::_Rb_tree_node_base*, std::_Rb_tree_node_base*, std::_Rb_tree_node_base&)'
/usr/include/c++/4.6/bits/stl_tree.h:203: error: undefined reference to 'std::_Rb_tree_decrement(std::_Rb_tree_node_base*)'
/usr/include/c++/4.6/iostream:75: error: undefined reference to 'std::ios_base::Init::Init()'
/usr/include/c++/4.6/iostream:75: error: undefined reference to 'std::ios_base::Init::~Init()'
nbee_link/libnbee_link.a(nbee_link.o)(.eh_frame+0x86d3): error: undefined reference to '__gxx_personality_v0'
collect2: ld returned 1 exit status
```

Maybe this link needs to be done with g++?

--- a/udatapath/automake.mk
+++ b/udatapath/automake.mk
@@ -46,6 +46,7 @@ udatapath_ofdatapath_SOURCES = \
 
 udatapath_ofdatapath_LDADD = lib/libopenflow.a oflib/liboflib.a oflib-exp/liboflib_exp.a nbee_link/libnbee_link.a $(SSL_LIBS) $(FAULT_LIBS)
 udatapath_ofdatapath_CPPFLAGS = $(AM_CPPFLAGS)
+nodist_EXTRA_udatapath_ofdatapath_SOURCES = dummy.cxx
 
 EXTRA_DIST += udatapath/ofdatapath.8.in
 DISTCLEANFILES += udatapath/ofdatapath.8

